README.md

```diff
@@ -66,11 +89,13 @@ Option | Description | Default | Valid options
 --- | --- | --- | ---
 Username | Replaces the default username of the webhook bot for the sent message only | Username unchanged | Any non-empty string (2-32 chars. inclusive)
 Author | Adds an author field to the header if set | Author not set | Any non-empty string (1-256 chars inclusive)
-DisableInlineFields | Inline means whether Discord will display the field in a column (with maximum three columns to a row). Setting this to `true` will cause Discord to display the field in its own row. | false | bool 
+DisableInlineFields | Inline means whether Discord will display the field in a column (with maximum three columns to a row). Setting this to `true` will cause Discord to display the field in its own row. | false | bool
```

Comment:
As specified above, the previous ordering should be kept.

Reply:
✔